.gitignore

```diff
@@ -1,4 +1,752 @@
-/node_modules/
-package-lock.json
-*.js
-*.d.ts
+### Eclipse ###
```

Comment:
This is a really big file. What is purpose of it? Let’s not add it.

Reply:
the package.lock.json should be added to repositories. See https://stackoverflow.com/a/44210813/6303600